### ISSUE TITLE
Modify 'hyphal tip' terms to 'hyphal tip cell'

### ISSUE THREAD

**Issue by dianeoinglis** [obophenotype/fungal-anatomy-ontology#14]
The term 'hyphal tip' is a cell part. It's the apex of a growing hypha where hyphal tip cell growth occurs. Cell parts are not currently included in the FAO. Either the 'hyphal tip' terms should be obsoleted and the new term 'hyphal tip cell' be created or all 'hyphal tip' terms be converted to 'hyphal tip cell.'

The following terms are affected:
0001011
0001015
0001016
0001017
0001019
0001020
0001021

**Comment by mah11** [comment]
I don't have a good sense of which approach (obsoleting or not) would be better, but if there's a use for the full set of "hyphal tip cell" terms it would be straightforward to convert the existing terms.

Any preference, @Achchuthan or @dianeoinglis ?

**Comment by Achchuthan** [comment]
I agree that hyphal tip is a cell part. As it is present in GO, hyphal tip terms can be made obsolete. I think the term "hyphal tip cell" would be inappropriate as hyphal tip is a part of the cell from which the hyphae elongates and it is not a distinct type of cell at the type of hyphae.
